CHANGELOG.md

```diff
@@ -68,3 +68,58 @@
 [#21]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/21
 [#22]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/22
 [#23]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/23
+
+# v2.0.0 (2024-06-20)
+
+## Kit Features
+
+* Move code to core kit ([#1])
+* Use Bottlerocket Settings SDK for settings models - ([#7])
+
+## OS Changes
+
+* Add mdadm packages for software RAID ([#4035]) - Thanks tzneal!
+* Kernel Updates ([#4049],[#4039], [#4005], [#3972], [#3976])
```

Comment:
Should we list which kernels and to what versions like we do for newer releases?

Reply:
That makes sense. Updated